src/html5_parser/tokenizer/mod.rs

```diff
@@ -2271,16 +2272,22 @@ impl<'a> Tokenizer<'a> {
     }
 
     // Sets the given name into the current token
-    fn set_name_in_current_token(&mut self, new_name: String) {
-        match &mut self.current_token.as_mut().unwrap() {
+    fn set_name_in_current_token(&mut self, new_name: String) -> Result<()> {
+        match &mut self.current_token.as_mut().expect("current token") {
```

Comment:
I don't think this should be an `expect`, now that I think about it.  I wasn't sure what to do in this case.